### PR TITLE
ETL changes to extract new question type

### PR DESCRIPTION
**Pull request by VineetBala-AOT**
*Description of changes:*
- Removed the insert to the tables response_type_radio, response_type_selectbox and response_type_textarea as these tables are no longer being used.
- Added new logic to extract and load select and survey type questions.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

**Review by jadmsaadaot — APPROVED** [review]

File: met-etl/src/etl_project/services/ops/submission_etl_service.py [hunk 1]
```diff
@@ -180,9 +177,12 @@ def _extract_submission(form_questions, met_survey, metsession, submission, mete
                 elif component_type == FormIoComponentType.CHECKBOX.value:
                     _save_checkbox(metetlsession, context, answer_key, component, etl_survey, user, submission,
                                    submission_new_runcycleid)
-                elif component_type == FormIoComponentType.TEXT.value:
-                    _save_text(metetlsession, context, etl_survey, component, answer_key, user, submission,
-                               submission_new_runcycleid)
+                elif component_type == FormIoComponentType.SELECTLIST.value:
+                    _save_select(metetlsession, context, answer_key, component, etl_survey, user, submission,
```
Comment (saravanpa-aot) [inline met-etl/src/etl_project/services/ops/submission_etl_service.py:181]: can you do another PR just by changing metetlsession to met_etl_session ?

**Review by saravanpa-aot — APPROVED** [review]

File: met-etl/src/etl_project/services/ops/submission_etl_service.py (hunk 1 above)
Comment (VineetBala-AOT) [reply met-etl/src/etl_project/services/ops/submission_etl_service.py:181]: Ok sure will do in another PR.
